assets/js/components/setup/CompatibilityChecks/CompatibilityErrorNotice.js

```diff
@@ -101,9 +103,14 @@ export default function CompatibilityErrorNotice( { error } ) {
 		case ERROR_API_UNAVAILABLE:
 			return (
 				<p>
-					{ __(
-						'Site Kit cannot access the WordPress REST API. Please ensure it is enabled on your site.',
-						'google-site-kit'
+					{ createInterpolateElement(
+						__(
+							'Site Kit cannot access the WordPress REST API. Please ensure it is enabled on your site. <GetHelpLink />',
+							'google-site-kit'
+						),
+						{
+							GetHelpLink: <GetHelpLink errorCode={ error } />,
+						}
```

Comment:
Why do we need to interpolate it? Why not to just display the get help link after the message?

```
<p>
    { __( 'Site Kit cannot access the WordPress REST API...', ... ) }
    <GetHelpLink errorCode={ error } />
</p>
```

Same for other cases.

Reply:
@tofumatt, when you spoke that `{ ' ' }` spaces are not localisable in this https://github.com/google/site-kit-wp/pull/5646#discussion_r937036716, what did you mean? A whitespace is not a punctuation and it has no different rules in different languages. Thus a whitespace is a whitespace in all languages.

I think having `{ __( ... ) }{ ' ' }<Tag ...>` is much better than using unnecessary `createInterpolateElement` call.

Reply:
The issue is that the _order_ of the components might matter in different locales. So some locales might want to place the text that appears before the whitespace in English _after_ the whitespace in a different language, because of grammar rules/sentence structure in different locales.

It might not apply to this instance (though it still could), but it's always best to never assume order in content when it can be localised.

Reply:
That makes sense. Thanks, @tofumatt 👍

Reply:
> the _order_ of the components might matter in different locales.

@tofumatt browsers take care of it as far as I know. 

> So some locales might want to place the text that appears before the whitespace in English _after_ the whitespace in a different language, because of grammar rules/sentence structure in different locales.

Can you show me an example? Do you mean that in some locales instead of `some text <a>tag</a>` they would want to set it to be ` some text<a>tag</a>` or I miss something?

Reply:
I just saw this now (I don't always see GitHub pings as I get too many 😓). But the idea is some locales might want to localise the text in a different order, eg:

`some text <a>tag</a>` could wind up being `<a>tag</a> some text`

This is because some languages have totally different sentence structure and will want to position a verb, action, or other text in a non-English order (Japanese is a good example).

So maximum flexibility is obtained when using the `createInterpolateElement` approach.